sympy/core/add.py

```diff
@@ -1222,7 +1215,16 @@ def __neg__(self):
             return super().__neg__()
         return Add(*[-i for i in self.args])
 
-add = AssocOpDispatcher('add')
+
+def add(*args, evaluate=False, **kwargs):
+    kwargs.update(evaluate=evaluate, _sympify=False)
```

Comment:
I wonder if this should be `args` rather than `*args`. I find the need to star-unpack everywhere quite annoying.

Reply:
Won't it be confusing to have different signatures between `Add` and `add`?

Reply:
Possibly. The reason it is this way is because `Add` needs to use `*args` because it's `Basic`. If you were writing a function for this in general though I don't think you'd use `*args` (e.g. like Python's `sum` or SymPy's `prod` functions).

Reply:
Sounds reasonable. I changed this.